source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs

```diff
@@ -568,6 +568,34 @@ public void GetPlayerStats(Action<CommonStatusCodes, PlayerStats> callback)
                     });
             }
         }
+        
+        ///<summary></summary>
+        /// <seealso cref="GooglePlayGames.BasicApi.IPlayGamesClient.ShowCompareProfileUI"/>
+        public void ShowCompareProfileUI(string userId, Action<UIStatus> callback)
+        {
+            if (!IsAuthenticated())
+            {
+                InvokeCallbackOnGameThread(callback, UIStatus.NotAuthorized);
+                return;
+            }
+
+            using (var playersClient = getPlayersClient())
+            using (var task = playersClient.Call<AndroidJavaObject>("loadPlayer", userId))
+            {
+                AndroidTaskUtils.AddOnSuccessListener<AndroidJavaObject>(
+                    task, annotatedData =>
+                    {
+                        using (var user = annotatedData.Call<AndroidJavaObject>("get"))
+                        {
+                            AndroidHelperFragment.ShowCompareProfileUI(user, AsOnGameThreadCallback(callback));
+                        }
+                    });
+                AndroidTaskUtils.AddOnFailureListener(task, exception =>
+                {
+                    InvokeCallbackOnGameThread(callback, UIStatus.NotAuthorized);
```

Comment:
UIStatus.NotAuthorized probably there are more cases it could faild and most frequent one will be no internet connection.
We should get status code from ApiException and switch over possible status codes